src/frontend/src/handler/mod.rs

```diff
@@ -204,16 +205,35 @@ pub async fn handle(
             returns,
             params,
         } => {
-            create_function::handle_create_function(
-                handler_args,
-                or_replace,
-                temporary,
-                name,
-                args,
-                returns,
-                params,
-            )
-            .await
+            if params.using.is_some() {
+                // User defined function with external source (e.g., language [ python / java ])
+                create_function::handle_create_function(
+                    handler_args,
+                    or_replace,
+                    temporary,
+                    name,
+                    args,
+                    returns,
+                    params,
+                )
+                .await
+            } else {
+                // SQL user defined function, the logic is mostly inlined
+                debug_assert!(
+                    params.using.is_none(),
+                    "`params.using` must be none for sql udf function"
+                );
```

Comment:
Is this ensured by sqlparser? 

And I expect a test for such ill-formed cases

Reply:
I guess not. In parser, `using` and `language` are parallel syntax elements.

Reply:
In general with sql udf, `using` must not appear (and that's why I put the debug assertion here), this is also conforming to pg's standard.